web/src/main/java/org/mskcc/oncotree/OncotreeApplication.java

```diff
@@ -63,7 +63,7 @@ private static ApiInfo apiInfo() {
             return apiInfoBuilder
                     .title("OncoTree API")
                     .description("OncoTree API definition from MSKCC")
-                    .version("1.0.0")
+                    .version("2.0.0")
```

Comment:
I believe the API is unchanged, so no change should be made here.

Reply:
The change was reverted, but it seems we still kept the copyright update in place. It seems incorrect to put a copyright year (2024) update when there was not actual change to the file. But maybe we should adopt a "single unified copyright message" which is comprehensive of changes to the entire codebase. Then attach that header to every file in the codebase. If we plan to do something like that, it would save us the effort of accounting for the different change history of any particular file and how the differ from each other.